packages/astro/src/core/build/index.ts

```diff
@@ -68,7 +68,6 @@ class AstroBuilder {
 			logging,
 		});
 		this.manifest = createRouteManifest({ settings: this.settings }, this.logging);
-
```

Comment:
I think we can undo this line removal, as it makes reading the code harder when compact.